lib/openid_connect/application.ex

```diff
@@ -0,0 +1,24 @@
+defmodule OpenIDConnect.Application do
+  use Application
+
+  def start(_type, _args) do
+    opts = [strategy: :one_for_one, name: FzVpn.Supervisor]
```

Comment:
this is still referencing a non-existing supervisor. What is `FzVPN`?

Reply:
@bcardarella Oh, I'm sorry, that code got here from our internal codebase. Fixed ✅